SECURITY.md

```diff
@@ -42,20 +42,25 @@ Certain members of the Magma team have been designated as the "vulnerability man
 
 5. Decide whether the report calls for a formal advisory in addition to an issue. 
 
-6. If it does not call for an advisory, inform the discloser, taking care not to reveal potential weaknesses, then prioritize and fix the issue according to ordinary workflows.
+  a. If it does not call for an advisory, inform the discloser, taking care not to reveal potential weaknesses, then prioritize and fix the issue according to ordinary workflows.
 
-If the report does merit an advisory, the vulnerability management team will:
+  b. If the report does merit an advisory, the vulnerability management team will execute the Advisory Procedure below
+  
+6.  Add the discloser to the security acknowledgements section in README.md. 
```

Comment:
As above: We should make this optional upon the wishes of the disclosing person.

Reply:
Added.